client/src/pages/feedback.tsx

```diff
@@ -116,16 +128,70 @@ const columnHeaders: ColumnDef[] = [
   },
 ];
 
+function FormatMentorQu(
```

Comment:
It is convention that functions are lowerCamelCase, so this should be formatMentorQu (why not formatMentorQuestions?)